dotnet-algorand-sdk/V2/Indexer/Model/Model.cs

```diff
@@ -58,7 +58,7 @@ public partial class Account
 
         /// <summary>\[teap\] the sum of all extra application program pages for this account.</summary>
         [Newtonsoft.Json.JsonProperty("apps-total-extra-pages", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
-        public int? AppsTotalExtraPages { get; set; }
+        public long? AppsTotalExtraPages { get; set; }
```

Comment:
Good example here - should int? really be long? for this??

Reply:
i agree.. we should find out somehow what is the real datatype in golang